src/Pages/Profile/admin/AdminDashboard.js

```diff
@@ -27,6 +44,9 @@ function AdminDashboard() {
             </Link>
           );
         })}
+        <Button onClick={handleButtonClick}>
```

Comment:
make this single component on its own page, where the user can only view it if they are an officer or an admin